_layouts/about.html

```diff
@@ -24,7 +24,7 @@
     <!-- /.photo-container -->
 
     <section class="ppl-container">
-      <div class="ppl-inner grid">
+      <div class="ppl-inner grid" id="aboutDirectors">
```

Comment:
think this might be from another pull request but would favour the covention of `about-directors` on ids